### ISSUE TITLE
Product versions list is not intuitive to users

### ISSUE THREAD

**Issue by Mariangela** [wso2/product-apim#3516]
**Description:**
The product versions list does not get updated properly when new API versions are created.

- I even check in a private window and also tried signing out and signing in again, but I had the same result.

- I also tried to make version 2.0.0 and 5.0.0 default versions at the time they were the newest versions to see if only the default versions are meant to be displayed but that too did not work.

**Suggested Labels:**
2.5.0, API Publisher

**Suggested Assignees:**

**Affected Product Version:**
2.5.0

**OS, DB, other environment details and versions:**  

Chrome, 2.5.0 RC4  

**Steps to reproduce:**
1. I created PhoneVerification API version 1.0.0, 2.0.0, 3.0.0, 4.0.0 and 5.0.0. 
2. When I click APIs> PhoneVerification 1.0.0 > Versions,  I only see that  PhoneVerification API has version 1.0.0, 2.0.0.

**Related Issues:**

![screen shot 2018-07-06 at 12 22 24 pm](https://user-images.githubusercontent.com/5195851/42364076-925fcd72-8117-11e8-8b3c-1bbee727ad7c.png)
![screen shot 2018-07-06 at 12 13 03 pm](https://user-images.githubusercontent.com/5195851/42364082-94ee54f0-8117-11e8-929e-f5c774bfc040.png)

**Comment by Mariangela** [comment]
On testing this issue further I found that only published and prototyped API versions get listed in this table, because when I tried publishing ver 5.0.0 it appeared in the table. This is not intuitive to a new user and they will get confused with this just like I did and think that it is a bug in the product.


It will be good if we can add a help text by saying something similar to the following.
 "Only published or prototyped APIs are displayed."


 
![screen shot 2018-07-06 at 12 27 53 pm](https://user-images.githubusercontent.com/5195851/42364333-64e13f4c-8118-11e8-9fc0-f6eeedd0db3c.png)
